lenskit-core/src/main/java/org/lenskit/data/dao/file/CollectionEntitySource.java

```diff
@@ -26,12 +26,14 @@
 import org.lenskit.util.io.ObjectStreams;
 
 import java.io.IOException;
+import java.io.Serializable;
 import java.util.Collection;
 
 /**
  * Entity source backed by a collection.
  */
-class CollectionEntitySource implements EntitySource {
+class CollectionEntitySource implements EntitySource, Serializable {
```

Comment:
I hope this interface is not specific to file because entity data might come from elsewhere.